dist/appform-field.tpl.html.js

```diff
@@ -7,6 +7,10 @@ try {
 
 ngModule.run(['$templateCache', function ($templateCache) {
   $templateCache.put('wfm-template/appform-field.tpl.html',
+    '<!--\n' +
+    ' Parent view for rendering all of the field types.\n' +
+    '-->\n' +
+    '\n' +
```

Comment:
For the moment, I think we do. But I will look into how it fits into the overall solution and see if it can be cleaned up.